app/models/katello/deb.rb

```diff
@@ -0,0 +1,82 @@
+module Katello
+  class Deb < Katello::Model
+    include Concerns::PulpDatabaseUnit
+
+    CONTENT_TYPE = Pulp::Deb::CONTENT_TYPE
+
+    has_many :repositories, :through => :repository_debs, :class_name => "Katello::Repository"
+    has_many :repository_debs, :class_name => "Katello::RepositoryDeb", :dependent => :destroy, :inverse_of => :deb
+
+    scoped_search :on => :name, :complete_value => true
+    scoped_search :on => :version, :complete_value => true
+    scoped_search :on => :architecture, :complete_value => true
+    scoped_search :on => :filename, :complete_value => true
+    scoped_search :on => :checksum
+
+    before_save lambda { |deb| deb.description = deb.description.truncate(255) unless deb.description.blank? }
+
+    def self.default_sort
+      order(:name).order(:epoch).order(:version_sortable).order(:release_sortable)
+    end
```

Comment:
This isn't being used i don't think?  (it errors when i try to use it in the rails console, due to epoch)